WordPress/src/main/java/org/wordpress/android/ui/posts/JetpackSecuritySettingsActivity.java

```diff
@@ -0,0 +1,59 @@
+package org.wordpress.android.ui.posts;
+
+import android.os.Bundle;
+import android.view.MenuItem;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import org.wordpress.android.R;
+
+public class JetpackSecuritySettingsActivity extends AppCompatActivity {
+    public static final int JETPACK_SECURITY_SETTINGS_REQUEST_CODE = 101;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.fragment_jetpack_security_settings);
+
+        setupToolbar();
+    }
+
+    private void setupToolbar() {
+        setTitle(getResources().getText(R.string.jetpack_security_setting_title));
+
+        Toolbar toolbar = findViewById(org.wordpress.mobile.ReactNativeGutenbergBridge.R.id.toolbar);
```

Comment:
Do we need the full package name here for the String?

Reply:
Ah no, thanks! 
Fixed.